src/Momento.Sdk.Incubating/Responses/CacheDictionaryGetFieldResponse.cs

```diff
@@ -10,24 +13,34 @@ public abstract class CacheDictionaryGetFieldResponse
     public class Hit : CacheDictionaryGetFieldResponse
     {
         protected readonly ByteString value;
+        protected readonly ByteString? field;
 
-        public Hit(_DictionaryGetResponse response)
+        public Hit(ByteString? field, _DictionaryGetResponse response)
```

Comment:
@malandis 
Here and other response types (Miss and Error), does `field` need to be nullable? or is there any other way to make this work?
I made `field` nullable because:
1. We want to check passed `fields` has at least one element in it (`fields.ToList().Count != 0`), if not `field` will be null.
2. Users could pass null to `fields` when they call `DicationaryGetFieldAsync`

As I write this, I might need to update `CacheDictionaryGetFieldsResponse` to take nullable fields as well to match...

Reply:
[Here](https://github.com/momentohq/client-sdk-dotnet-incubating/pull/87#discussion_r1066471756) @cprice404 had a similar question as well.

Reply:
Once we get to this point in the code, `field` cannot be nullable. We check for null fields in the top-level API. See https://github.com/momentohq/client-sdk-dotnet/blob/main/src/Momento.Sdk/SimpleCacheClient.cs#L71-L86

Reply:
So no need to check [this](https://github.com/momentohq/client-sdk-dotnet-incubating/pull/87#discussion_r1067253840) then?